idd/Energy+.idd.in

```diff
@@ -526,18 +540,18 @@ ShadowCalculation,
        \note only really applicable to RunPeriods
```

Comment:
This line is misleading, because SizingPeriod:WeatherFileDays would also use this. I'd probably just delete this note

For the notes above this line . . . 
AverageOverDaysInFrequency --> Periodic
~0=Use Default Periodic Calculation~ (minimum is 1, so zero isn't valid since who know when).

Reply:
Agreed. This is all existing text, though I can clean it up here.